src/fidesops/analytics.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import os
+from platform import system
+from typing import Dict
+
+from fideslog.sdk.python.client import AnalyticsClient
+from fideslog.sdk.python.event import AnalyticsEvent
+from fideslog.sdk.python.exceptions import AnalyticsError
+from fideslog.sdk.python.utils import FIDESOPS, generate_client_id
+
+from fidesops.core.config import config, update_config_file
+
+from ._version import get_versions
+
+logger = logging.getLogger(__name__)
+
+
+def get_version() -> str:
+    """Version of Fidesops"""
+    return get_versions()["version"]
+
+
+def in_developer_mode() -> bool:
+    """True if running in dev mode"""
+    return config.dev_mode
+
+
+def in_docker_container() -> bool:
+    """`True` if the command was submitted within a Docker container. Default: `False`."""
+    return True
+
+
+def running_on_local_host() -> bool:
+    """For events submitted as a result of making API server requests, `True` if the API server is running on the user's local host. Default: `False`."""
+    return False
```

Comment:
Here we need a way to detect whether the server is running on `localhost` or equivalent (`0.0.0.0`, `127.0.0.1`, etc). Passing `False` for all events would skew the analytics.

Reply:
So technically we always run "locally" as in local to the docker instance. But perhaps in the future we'll have a hosted version. - https://github.com/ethyca/fidesops/blob/main/src/fidesops/main.py#L62

Reply:
I see... So this isn't straightforward, but maybe it's best to always return `True` if we're always running on `0.0.0.0` ? It's your call.